lib/model/Physical.mjs

```diff
@@ -1,85 +1,140 @@
+/**
+ * A fixture's technical data, refering to the hardware and not the DMX protocol.
+ */
 export default class Physical {
+  /**
+   * Creates a new Physical instance.
+   * @param {!object} jsonObject A fixture's or mode's physical JSON data.
+   */
   constructor(jsonObject) {
-    this.jsonObject = jsonObject; // calls the setter
-  }
-
-  set jsonObject(jsonObject) {
     this._jsonObject = jsonObject;
   }
 
-  // used to combine with a mode's physical override
+  /**
+   * @returns {!object} The object from the JSON data that is represented by this Physical object.
+   */
   get jsonObject() {
     return this._jsonObject;
   }
 
-
+  /**
+   * @returns {?Array.<number>} Width, height and depth of the fixture in millimeters. Defaults to null.
+   */
   get dimensions() {
     return this._jsonObject.dimensions || null;
   }
 
+  /**
+   * @returns {?number} Width of the fixture in millimeters. Defaults to null.
+   */
   get width() {
     return this.dimensions !== null ? this.dimensions[0] : null;
   }
 
+  /**
+   * @returns {?number} Height of the fixture in millimeters. Defaults to null.
+   */
   get height() {
     return this.dimensions !== null ? this.dimensions[1] : null;
   }
 
+  /**
+   * @returns {?number} Depth of the fixture in millimeters. Defaults to null.
+   */
   get depth() {
     return this.dimensions !== null ? this.dimensions[2] : null;
   }
 
+  /**
+   * @returns {?number} Weight of the fixture in kilograms. Defaults to null.
+   */
   get weight() {
     return this._jsonObject.weight || null;
   }
 
+  /**
+   * @returns {?number} Power consumption of the fixture in Watt. Defaults to null.
+   */
   get power() {
     return this._jsonObject.power || null;
   }
 
+  /**
+   * @returns {?string} The DMX plug to be used to control the fixture, e.g. "3-pin" (XLR). Defaults to null.
+   */
   get DMXconnector() {
     return this._jsonObject.DMXconnector || null;
   }
 
 
+  /**
+   * @returns {?boolean} Whether there is physical data about the light source available.
```

Comment:
Sounds very German :wink: Better: "Whether physical data about the light source is available."